pages/project.py

```diff
@@ -39,10 +39,12 @@ class ProjectPage(GuidBasePage):
     make_private_link = Locator(By.XPATH, '//a[contains(text(), "Make Private")]')
     confirm_privacy_change_link = Locator(By.XPATH, '//a[text()="Confirm"]')
     cancel_privacy_change_link = Locator(By.XPATH, '//a[text()="Cancel"]')
-    collections_container = Locator(By.CLASS_NAME, 'collections-container')
-    collections_link = Locator(
+    collections_container = Locator(
+        By.CSS_SELECTOR, '#projectBanner > div.row > div.collections-container.col-12'
```

Comment:
Looks like you are using this locator to assert the container has loaded. Would it be easier/cleaner to just use the div ID than to traverse down the CSS selectors? 

`collections_container = Locator(By.ID, 'collections-header')`